lib/govuk_index/publishing_event_worker.rb

```diff
@@ -39,9 +39,11 @@ def perform(messages)
         process_action(processor, routing_key, payload)
       end
 
-      response = processor.commit
-      process_response(response, messages) if response
+      responses = processor.commit
 
+      (responses || []).each do |response|
+        process_response(response, messages)
```

Comment:
Does this make the `if response['items'].count > 1` check in `process_response` redundant?

Reply:
No